src/ClientHandle.cpp

```diff
@@ -1863,6 +1863,12 @@ void cClientHandle::RemoveFromWorld(void)
 	m_LastStreamedChunkZ = 0x7fffffff;
 
 	m_HasSentPlayerChunk = false;
+
+	// Do not send chunks for 1.5 seconds.
+	// Apparently, the chunks of the old world fail to unload if
+	// chunks in the new world which have the same coords are sent too fast after the unload.
+	// this is a temp fix, find the root cause!
+	m_ChunkSendDelay = 30;
```

Comment:
Without this PR, when teleporting to a world of the same type and to the same coords, the chunks of the old world remain visible, until they are slowly replaced by the chunks of the new world, despite the fact that Cuberite tells the client to unload those chunks. Waiting a bit prior to sending the new chunks resolves it.